plugins/statistics.koplugin/main.lua

```diff
@@ -1033,6 +1033,56 @@ The max value ensures a page you stay on for a long time (because you fell aslee
                             },
                         },
                     },
+                    {
+                        text_func = function()
+                            -- @translators %1 is the hour and %2 is the minute, units for h and m will be automatically added
+                            return T(_("Calendar days start at %1 %2"),
+                                T(C_("Time", "%1h"), self.settings.calendar_day_start_hour or 0),
+                                T(C_("Time", "%1m"), self.settings.calendar_day_start_minute or 0)
+                            )
+                        end,
+                        callback = function(touchmenu_instance)
+                            local DoubleSpinWidget = require("/ui/widget/doublespinwidget")
+                            local getExtraButtonText = function()
+                                return self.settings.calendar_day_start_time_limit_use and _("Tap to use in calendar and daily timeline") or _("Tap to use only in daily timeline")
```

Comment:
Feels a bit odd for the UX, this wording and this button not really able to show the current state.
May be it's just best to add that as a checkbox in the statistics `Settings>` submenu - we have room.
Also, `Calendar days start at 00:00` and this `[x] Also use in calendar view` may get some separators and appear below the month view settings (even if `[x] Also use in calendar view` is related to the monthly view ?:)
Feels a bit odd this appears in the middle of the strictly month stuff.

edit: tested again, and yes, we need this to be really visible, so one knows his monthly view if fucked up and knows how to unfuck it :)
I also think this "also shift day start in calendar view" shouldn't be the default.